src/main/resources/assets/actuallyadditions/lang/en_US.lang

```diff
@@ -616,10 +639,18 @@ info.actuallyadditions.booklet.manualName.1.6=Actually Addons
 info.actuallyadditions.booklet.manualName.1.7=Additional Addons
 info.actuallyadditions.booklet.manualName.2=Manual
 info.actuallyadditions.booklet.edition=Edition
+info.actuallyadditions.booklet.searchField=Click to search...
 info.actuallyadditions.deathRecorded=Your death has been recorded. Use a Death Tracker to find the death location!
 info.actuallyadditions.placer.sides.1=Placement Sides
 info.actuallyadditions.placer.sides.2=Usually when placing down blocks, they are placed towards the side of another block that you are looking at. Because the Phantom Placer can place blocks in mid-air, it doesn't know that. Usually you should just set it to a solid side.
 info.actuallyadditions.machineBroke=The block shattered as you tried to break it. Maybe it wasn't sturdy enough to be broken?
+info.actuallyadditions.energyUnit=Cristal Flux
+info.actuallyadditions.energyDisplay=%s/%s Cristal Flux
+info.actuallyadditions.redstoneMode.name=Redstone Mode: %s
```

Comment:
The way this is formatted should also be hardcoded, so that the string "Redstone Mode" is localized, and then a colon is added, followed by the localized pulse/deactivation string.